cli/tests/unit/test_semver_matching.py

```diff
@@ -0,0 +1,27 @@
+import pytest
+from semdep.package_restrictions import semver_matches
+
+@pytest.mark.quick
+@pytest.mark.parametrize(
+    "expression,candidate,match",
+    [
+        pytest.param(">1.0.0", "1.0.1", True),
+        pytest.param(">1.0.0", "2.0.0", True),
+        pytest.param(">=1.0.0", "1.0.0", True),
+        pytest.param(">1.0.0", "0.1.0", False),
+        pytest.param(">1.0.0", "0.1.0-beta", False),
+        pytest.param("<1.0.1", "1.0.1", False),
+        pytest.param("<1.0.1", "1.0.0", True),
+        # pytest.param("<1.0.1", "0.1.0-beta", True),
```

Comment:
Could xfail this instead of commenting it out? Because in the ideal case it would work. For that you do need `pytest.param`, it's
```
pytest.param("<1.0.1", "0.1.0-beta", True, marks=pytest.mark.xfail)
```
See https://docs.pytest.org/en/7.1.x/how-to/skipping.html#skip-xfail-with-parametrize